app/views/components/_blue-box-campaign.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="app-c-native-blue-box-campaign__border">
```

Comment:
I'd suggest renaming this - although it does include border styling, this could be changed in future. It's more of a `app-c-native-blue-box-campaign__container` or `app-c-native-blue-box-campaign__wrapper`